Formula/gnu-cobol.rb

```diff
@@ -1,8 +1,8 @@
 class GnuCobol < Formula
-  desc "Implements much of the COBOL 85 and COBOL 2002 standards"
-  homepage "https://sourceforge.net/projects/open-cobol/"
-  url "https://downloads.sourceforge.net/project/open-cobol/gnucobol/2.2/gnucobol-2.2.tar.xz"
-  sha256 "dc18fc45c269debfe86a4bbe20a7250983cba6238ea1917e135df5926cd024a0"
+  desc "COBOL85-202x compiler supporting lots of dialect specific extensions"
+  homepage "https://www.gnu.org/software/gnucobol/"
+  url "https://alpha.gnu.org/gnu/gnucobol/gnucobol-3.1-rc1.tar.xz"
```

Comment:
Is rc1 a stable release?

Reply:
At least more stable than everything that was before :-)
Especially more stable than old 2.2 (with its CVEs).

Reply:
Insecure and stable are two very distinct problems. If there is no stable version and the current version is a security problem I think we should disable the formula until a secure stable version is available.

Reply:
GC 2.2 is as insecure as old unpatched GCC versions are - nothing that breaks your system if you don't compile special crafted sources, and if done: mostly kills the executable itself...

If the "contains rc" is a hard rule then then package will stay "outdated" for likely 1-2 months more, it is not my decision for or against the policy that is normally enforced with the CI checks.

The main point _for_ the change now is: GnuCOBOL 3.1 RC 1 _is not_ a "final" release version, but _it is_ a tagged official version which is more stable than every version published before and is downward compatible.

Reply:
The things we ship have to be deemed stable by upstream. I'd be happy to merge this in a month with a stable version though.

https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
"deemed stable by upstream" - What a co-incidence, it is, see the [NEWS entry for the rc](https://savannah.gnu.org/forum/forum.php?forum_id=9766) :-) More questions from my side for this bottle in the PR discussion...